package.json

```diff
@@ -5,15 +5,15 @@
   "author": "Firebase (https://firebase.google.com/)",
   "license": "Apache-2.0",
   "scripts": {
-    "bootstrap": "lerna bootstrap --no-ci",
-    "lint": "lerna run lint",
-    "test": "lerna run test",
-    "compile": "lerna run compile --concurrency=8"
+    "bootstrap": "pnpm recursive install",
+    "lint": "pnpm recursive run lint --workspace-concurrency=4",
```

Comment:
Seems like from your testing, concurrency of 8 was faster than 4 (3.5 minutes vs 4.25 minutes). Is there another reason to use 4?

Reply:
On my MacBook (and I assume yours) 8-concurrency basically locks up the whole machine and turns it into an airplane :-)

Reply:
Haha in that case, 4 sounds good to me!